wordpress-mobile/WordPress-Android
# Populate quick start cards with tasks

Fixes #13782

This PR creates a repository which provides a live endpoint of the quick start items shown on the my site screen. There is also a new builder which takes these items and transforms them into the UI items. I've added the `start` method call from the main activity which is now passed to the improved my site fragment and renamed the Dummy classes into normal names. Everything should be tested. 

To test:
- Open the app with the my site flag turned on
- Notice the cards are now visible and populated (you might have to create a new site for this)
- Notice the "Create a site" item is marked as "done" and the progress has shifted

![Screenshot_1610715111](https://user-images.githubusercontent.com/1079756/104729455-bf0cbc00-5738-11eb-9723-8e91c4973829.png)


PR submission checklist:
